Softeq.XToolkit.Chat.iOS/ViewControllers/AddContactsViewController.cs

```diff
@@ -67,6 +67,7 @@ protected override void DoAttachBindings()
             TableViewSearchBar.TextChanged += TableViewSearchBarTextChanged;
             _tableViewSource.ItemTapped += TableViewSourceItemTapped;
             _tableViewSource.LastItemRequested += TableViewSourceLastItemRequested;
+            ViewModel.PaginationViewModel.Items.CollectionChanged += ItemsCollectionChanged;
```

Comment:
TODO:
- add `HasResults` property to the `PaginationViewModel` will be raised after changing `Replace/Add Range` collection
- bind to this property on iOS & Android